tools/codegen/api/lazy.py

```diff
@@ -58,8 +58,7 @@ def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, L
         raise AssertionError(f"unrecognized type {repr(typ)}")
 
 
-def isValueType(typ: Union[Type, BaseCType, OptionalCType, ConstRefCType, MutRefCType,
-                           ListCType, ArrayRefCType, ArrayCType, VectorCType, TupleCType]) -> bool:
+def isValueType(typ: Union[Type, CType]) -> bool:
```

Comment:
Looks like Will has a PR out to change this too https://github.com/pytorch/pytorch/pull/72756/files?diff=split&w=1#r808407558 (@wconstab )

Reply:
Yup, this will go away if Will's merge goes in before mine